src/main/java/org/isf/settings/model/Setting.java

```diff
@@ -0,0 +1,152 @@
+/*
```

Comment:
There are tests for this file that only cover 27% of the methods and 27% of the lines of code.  I would suggest more tests are needed.